packages/front-end/components/Modal.tsx

```diff
@@ -12,6 +13,7 @@ type ModalProps = {
   cta?: string;
   closeCta?: string;
   ctaEnabled?: boolean;
+  docsLink?: string;
```

Comment:
I think the code would be simpler if the prop was the id of the doc section instead of the full href.  That way, we wouldn't need to use the `getDocLink` function anywhere.  We just pass in the id to this component and it renders the `DocLink` component.